cinema_service/settings.py

```
@@ -9,6 +9,7 @@
 For the full list of settings and their values, see
 https://docs.djangoproject.com/en/4.0/ref/settings/
 """
+import os
```

Currently you import os directly. Consider adding a small helper or using `os.environ.get()` semantics rather than relying on `os.environ[...]` everywhere so you can provide defaults or raise a clear, descriptive error if a variable is missing. That will avoid cryptic KeyError traces when the environment is not configured exactly as expected and will make the app more robust in CI and local runs.